Application/EdFi.Ods.Api/Security/Authorization/AuthorizationSegmentSqlProviderBase.cs

```diff
@@ -46,6 +47,22 @@ public AuthorizationSegmentSqlProviderBase(IAuthorizationViewsProvider authoriza
             var segmentStatements = new List<string>();
             var parameters = new List<DbParameter>();
 
+            authorizationSegments.ToList().ForEach(eachSegment =>
+            {
+                eachSegment.ClaimsEndpoints.ToList().ForEach(eachClaimEndpoint =>
+                {
+                    if (EducationOrganizationEntitySpecification.IsEducationOrganizationIdentifier(eachClaimEndpoint.Name))
+                    {
+                        eachClaimEndpoint.Name = "EducationOrganizationId";
+                    }
+                    
+                    if (!PersonEntitySpecification.IsPersonIdentifier(eachSegment.SubjectEndpoint.Name))
```

Comment:
This story is very complex one for me .  What I know is I need to set every ClaimEndpoint which of EducationOrganizationEntitySpecification.IsEducationOrganizationIdentifier exists  to "EducationOrganizationId" and set every non person subjectendpoint to "EducationOrganizationId" so that it will be like EducationOrganizationIdToEducationOrganizationId tuple table to make query for authorization